csaf/src/rolie/mod.rs

```diff
@@ -0,0 +1,44 @@
+mod roliefeed;
+
+use crate::rolie::roliefeed::RolieFeed;
+use async_trait::async_trait;
+use url::Url;
+use walker_common::changes::{ChangeEntry, ChangeSource, Error};
+use walker_common::fetcher::Fetcher;
+
+#[async_trait(?Send)]
```

Comment:
Technically that works and seems like a clever way to inject that data. However, it feels like we are mixing two concepts here. I don't think the idea of the "ChangeSource" makes sense here. One is a ROLIE feed, while the other one is a `change.csv`. It looks like those two concepts don't mix well. So we should use individual concepts.